packages/middleware-endpoint/src/types.ts

```diff
@@ -40,3 +41,11 @@ export interface ContextParamInstruction {
   type: "contextParams";
   name: string; // The input structure's member name that has contextParams trait
 }
+
+/**
+ * @internal
+ */
+export interface OperationContextParamInstruction {
+  type: "operationContextParams";
+  get(input: any): any;
```

Comment:
can I assume that the path expression is solely based on the input? No other parameters are needed, right?

Reply:
Yes. As per internal documentation
> The `operationContextParams` trait defines one or more context parameters that **MUST** be bound to values specified in the operation input using a JMESPath expression